spec/twitter-ads/creative/video_spec.rb

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2015 Twitter, Inc.
+
+require 'spec_helper'
+
+describe TwitterAds::Creative::Video do
+
+  before(:each) do
+    stub_fixture(:get, :accounts_all, "#{ADS_API}/accounts")
+    stub_fixture(:get, :accounts_load, "#{ADS_API}/accounts/2iqph")
+  end
+
+  let(:client) do
+    Client.new(
+      Faker::Lorem.characters(15),
+      Faker::Lorem.characters(40),
+      "123456-#{Faker::Lorem.characters(40)}",
+      Faker::Lorem.characters(40)
+    )
+  end
+
+  let(:account) { client.accounts.first }
+
+  # check model properties
+  subject { described_class.new(account) }
+
+  read = %w(
+    id
+    tweeted
+    ready_to_tweet
+    duration
+    reasons_not_servable
+    preview_url
+    created_at
+    updated_at
+    deleted
+  )
+
+  write = %w(title description video_media_id)
+
+  include_examples 'object property check', read, write
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.